From 9932dc7e52f756b232afc9c19f5c7a290588032e Mon Sep 17 00:00:00 2001
From: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Date: Wed, 24 May 2023 09:47:53 +0530
Subject: [PATCH] Remove libgbm creation twice

Remove the libgbm creation as it is already created in
hardware/intel/external/mesa3d-intel/src/gbm/Android.mk

Tracked-On: NA
Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
---
 Android.bp | 34 ----------------------------------
 1 file changed, 34 deletions(-)

diff --git a/Android.bp b/Android.bp
index 9eb75e5..52ac7bd 100644
--- a/Android.bp
+++ b/Android.bp
@@ -139,40 +139,6 @@ cc_defaults {
     srcs: [":minigbm_gralloc0_files"],
 }
 
-cc_library {
-    name: "libgbm",
-    defaults: ["minigbm_defaults"],
-    host_supported: true,
-
-    srcs: [
-        ":minigbm_core_files",
-        "gbm.c",
-        "gbm_helpers.c",
-    ],
-
-    target: {
-        host: {
-            // Avoid linking to another host copy of libdrm; this library will cause
-            // binary GPU drivers to be loaded from the host, which might be linked
-            // to a system copy of libdrm, which conflicts with the AOSP one
-            allow_undefined_symbols: true,
-            header_libs: ["libdrm_headers"],
-        },
-        android: {
-            shared_libs: [
-                "libdrm",
-                "liblog"
-            ],
-        },
-    },
-    apex_available: [
-        "//apex_available:platform",
-        "com.android.virt",
-    ],
-
-    export_include_dirs: ["."],
-}
-
 // Generic
 cc_library_shared {
     name: "libminigbm_gralloc",
-- 
2.17.1

